CONTRIBUTING.md

```diff
@@ -63,16 +66,7 @@ Now `~/Projects/owning-a-home/node_modules/cf-buttons` will be a symlink pointin
 
 After changes have been made to components, nothing will be published to npm until
 those changes enter the `master` branch.
+Release management is now handled by [hubot-capital-framework](https://github.com/cfpb/hubot-capital-framework).
 
-1. Go to wherever cf is cloned locally: `cd capital-framework`
-1. Ensure you're on canary: `git checkout canary`
-1. Ensure you're up-to-date: `git pull upstream canary`
-1. Open `/CHANGELOG.md` and check the "Unreleased" section. For each component
-that's been modified, open its `/src/COMPONENT/package.json` and increment its
-version. Follow [SemVer](http://semver.org/) and increment either its major, minor,
-or patch version dependent upon the changes that were made to the component.
-1. Don't bump CF's primary `/package.json`. Travis will automatically increment its version.
-1. Delete any empty sections ("Added", "Changed", "Removed") from the "Unreleased" section.
-1. `git commit -am "Preparing release"`
-1. `git push upstream canary`
-1. Go to https://github.com/cfpb/capital-framework and open a pull request to merge `canary` into `master`.
+1. In a public chat room say "cfpbot capital framework prepare release"
```

Comment:
Minor suggestion: "In a room where cfpbot is present (preferably the FEWD channel), say …"